WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -261,10 +256,24 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) {
+        // We are currently supporting WPCom credentials only
+        // Update this when handling store credentials authentication.
+        guard let wpcomLogin = credentials.wpcom else {
+            return
+        }
+
+        /// Jetpack is required. Present an error if we don't detect a valid installation for a self-hosted site.
+        if let site = currentSelfHostedSite,
```

Comment:
Would it make sense to use `guard` instead of `if` to follow the existing pattern in this method? i.e. Check condition and return early with an error.

Reply:
Oh, I understand. Thanks for the info. 🙇